lib/js/components/DatePickers/DateRangePicker/DateRangePicker.vue

```diff
@@ -1,17 +1,15 @@
 <template>
 	<div ref="dateRangePickerRef" class="ds-dateRangePicker">
-		<date-picker-box
+		<date-box
 			:is-interactive="isInteractive"
 			:placeholder="placeholder"
 			:start-date="startDate"
-			:end-date="endDateIfDifferentThanStartDate"
+			:end-date="endDate"
 			:start-icon="startIcon"
 			:end-icon="endIcon"
 			:are-icons-hidden-on-mobile="areIconsHiddenOnMobile"
 			:state="state"
 			:color="color"
-			:start-date-eyebrow-text="eyebrowText"
-			:end-date-eyebrow-text="endDateEyebrowText"
 			:is-open="isOpen"
```

Comment:
should we rename it to `is-hoverable` or something like that? when we pass isOpen we only change background color on hover and icon color

Reply:
We actually block the change of a background color to hover specific.

Reply:
but the name `is open` means that something is open in this component - in this situation we do not open anything, we only turn hovering on or off - so the name is misleading

Reply:
`is-hoverable` would mean that hover state is completely disabled or enabled, but the pointer is always the same. I can change it on more meaningful name